src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPart.SummaryAdmin.cshtml

```diff
@@ -0,0 +1,8 @@
+@model OrchardCore.ContentManagement.Display.ViewModels.ContentItemViewModel
+
+@if (Model.ContentItem.Content["ListPart"] != null)
+{
+    <a class="btn btn-sm text-secondary" data-toggle="tooltip" title="@T["List Items"]" asp-route-action="List" asp-route-area="OrchardCore.Contents" asp-route-contentTypeId="@Model.ContentItem.ContentType">
+        <i class="fa fa-list"></i>
```

Comment:
Much better @hishamco 

Suggestions:
- Don't null check this, it can't get there if the driver didn't driver it there.
- Needs to be `admin-for` - to achieve the same functionality as clicking the underlined link would normally do.
- Needs text `List Items` not just an icon, lose the tooltip

Reply:
> Don't null check this, it can't get there if the driver didn't driver it there.

You are right, I forgot it's a list ;)

> Needs text List Items not just an icon, lose the tooltip

I just follow the same style for Edit, View icons